src/web/components/SharingPermission/SharingPermissionsTable.tsx

```diff
@@ -230,6 +230,7 @@ export function SharingPermissionsTable({
   const { sites, isLoading } = useAllSitesList();
   const getSharingParticipants: () => SharingSiteWithSource[] = () => {
     return sites!
+      .filter((p) => p.canBeSharedWith || sharedSiteIds.includes(p.id))
```

Comment:
Unfortunately in this case, we do need all sites - because they might be explicitly sharing with a participant that's not in the available list.
The rest of sharing uses the `useAvailableSiteList` call, but that could omit sites we need here.

The whole design of this section is a bit too complicated but I want to get this bug fixed without re-working it.